ext-plugin.fbs

```diff
@@ -100,6 +102,8 @@ table Rewrite {
     // To delete a request header, use a TextEntry without Value
     // To change the upstream host, use a TextEntry with Name equals to `host`
     headers:[TextEntry];
+    // To change the upstream response header
+    resp_headers:[TextEntry];
```

Comment:
Please add new field after the `args` to keep the offset.

Reply:
Ok, I'll correct it right away.